Resources/AzureDevOps/azure-pipelines.yml

```diff
@@ -0,0 +1,33 @@
+variables:
+  SolutionBaseName: 'NCS.DSS.ChangeFeedListener'
+  BuildPlatform: 'any cpu'
+  BuildConfiguration: 'release'
+
+steps:
+ - task: gittools.gitversion.gitversion-task.GitVersion@4
+   displayName: GitVersion
+   inputs:
+     preferBundledVersion: false
+ - task: NuGetToolInstaller@0
+   displayName: 'Use NuGet 4.3.0'
+ - task: NuGetCommand@2
+   displayName: 'NuGet restore ${{ variables.SolutionBaseName }}.sln'
+   inputs:
+     restoreSolution: '${{ variables.SolutionBaseName }}.sln'
+ - task: DotNetCoreCLI@2
+   displayName: 'dotnet build'
+   inputs:
+     command: 'build'
+     projects: '**/NCS.DSS.ChangeFeedListener.csproj'
+ - task: CopyFiles@2
```

Comment:
Has this artefact been deployed?  Typically it would need zipping up to be deployed to a function app.  One way to test the artefact is to download and deploy it to a function app in your own tenant